src/containers/Confirm/Address/index.tsx

```diff
@@ -0,0 +1,304 @@
+import cr from 'classnames';
+import * as countries from 'i18n-iso-countries';
+import * as React from 'react';
+import { Button } from 'react-bootstrap';
+import { InjectedIntlProps, injectIntl } from 'react-intl';
+import { connect, MapDispatchToPropsFunction } from 'react-redux';
+import { RouterProps } from 'react-router';
+import { withRouter } from 'react-router-dom';
+import { compose } from 'redux';
+import { languages } from '../../../api/config';
+import { CustomInput, UploadFile } from '../../../components';
+import { DropdownComponent } from '../../../components/Dropdown';
+import {
+    alertPush,
+    RootState,
+    selectCurrentLanguage,
+    selectSendAddressesSuccess,
+    sendAddresses,
+} from '../../../modules';
+
+interface ReduxProps {
+    lang: string;
+    success?: string;
+}
+
+interface DispatchProps {
+    sendAddresses: typeof sendAddresses;
+    fetchAlert: typeof alertPush;
+}
+
+interface State {
+    country: string;
+    address: string;
+    addressFocused: boolean;
+    city: string;
+    cityFocused: boolean;
+    postcode: string;
+    postcodeFocused: boolean;
+    fileScan: File[];
+}
+
+type Props = ReduxProps & DispatchProps & RouterProps & InjectedIntlProps;
+
+class AddressComponent extends React.Component<Props, State> {
+    public state = {
+        country: '',
+        address: '',
+        addressFocused: false,
+        city: '',
+        cityFocused: false,
+        postcode: '',
+        postcodeFocused: false,
+        fileScan: [],
+    };
+
+    public UNSAFE_componentWillReceiveProps(next: Props) {
+        if (next.success) {
```

Comment:
please, write ```if (!this.props.success && next.success)``` because action this.props.history.push can be dispatched few times